classes/PluginInstall.class.php

```diff
@@ -45,7 +45,7 @@ public function create_menu()
 		$this->page = add_submenu_page(
 			'plugins.php',
 			esc_html__('Install ClassicPress plugins', 'classicpress-directory-integration'),
```

Comment:
Plugins here should also be capitalized:
```
esc_html__(' Install ClassicPress Plugins',
```

Reply:
I thought I fixed the lowercase Plugin.

Reply:
Done!